src/dotnet-interactive-vscode-insiders/README.md

```diff
@@ -2,31 +2,83 @@ This extension is currently **_in preview_**.
 _The Visual Studio Code notebook support that this extension uses is also in preview and design is ongoing, so the extension might not work._
 ---
 
-# .NET Interactive Notebooks
+# Polyglot Notebooks
+
+With the Polyglot Notebooks extension, you can use multiple languages in the same notebook and share variables between them. No more installing different Jupyter kernels, using wrapper libraries, or different tools to get the best language server support for the language of your choice. Always use the best language for the job and seamlessly transition between different states of your workflow, all within one notebook. 
+
+## Languages Supported
+
+- C#
+- F# 
+- PowerShell
+- JavaScript
+- SQL
+- KQL (Kusto Query Language)
+- HTML*
+- Mermaid*
+
+*Variable sharing not available
+
+## Features
+
+- Connect to and query from Microsoft SQL Server databases and Kusto clusters
+- Run and execute code for all 8 featured languages
+- Language server support such as completions, syntax highlighting, and signature help for each language
+- Variable sharing between languages
+- Variable Explorer to check value and state of variables 
+- Create detailed diagrams and visualizations using [Mermaid](https://mermaid-js.github.io/mermaid/#/)
```

Comment:
Might be good to mention that other visualization libraries can be used as well.

Reply:
Any in particular you think are worth highlighting?